astropy/config/configuration.py

```diff
@@ -92,6 +92,12 @@ class Conf(_config.ConfigNamespace):
                 aliases=['astropy.utils.console.USE_COLOR'])
         conf = Conf()
     """
+
+    rootname = 'astropy'
```

Comment:
This is defined again in `ConfigItem`. Is this necessary here? Seems weird to define this twice separately.

Reply:
Different bits of the machinery use `ConfigItem` and `ConfigNamespace` I could not see a simple way of sharing the value.

Reply:
although maybe this isn't actually used anywhere... 😕